resources/visual_studio_code_dark_plus.xml

```diff
@@ -771,6 +783,13 @@
         <option name="FOREGROUND" value="39c8b0" />
       </value>
     </option>
+    <option name="WARNING_ATTRIBUTES">
```

Comment:
That's not JavaScript-specific change.

Reply:
I agree. I think I can remove that here and do a separate PR with warning styles.

Reply:
Yeah. Please do. Thanks.